### PR TITLE
Backport of "Remove Marketplace AMI builds, plus other tidying (#4502)"

### PR DESCRIPTION
**Pull request by webvictim**
Backport of #4502

### CODE REVIEW

File: examples/aws/terraform/starter-cluster/README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ TF_VAR_license_path ?="/path/to/license"
 # OSS: aws ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-oss*'
 # Enterprise: aws ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-ent*'
 # FIPS 140-2 images are also available for Enterprise customers, look for '-fips' on the end of the AMI's name
-TF_VAR_ami_name ?="gravitational-teleport-ami-ent-4.2.9"
+TF_VAR_ami_name ?="gravitational-teleport-ami-ent-4.3.7"
```
Comment (benarent) [inline examples/aws/terraform/starter-cluster/README.md:85]:
```suggestion
TF_VAR_ami_name ?="gravitational-teleport-ami-ent-4.0.0"
```

File: examples/aws/terraform/AMIS.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Public Teleport AMI IDs
+
+For your convenience, this is a list of public Teleport AMI IDs which are published by Gravitational. This list
+is updated when new AMI versions are released.
+
+### OSS
```
Comment (benarent) [inline examples/aws/terraform/AMIS.md:6]: Do these need to be bumped to 4.0.0?

**Comment by webvictim** [comment]
This should all get bumped when I merge/rebase

File: examples/aws/terraform/ha-autoscale-cluster/README.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ export TF_VAR_cluster_name="teleport.example.com"
 # OSS: aws ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-oss*'
 # Enterprise: aws ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-ent*'
 # FIPS 140-2 images are also available for Enterprise customers, look for '-fips' on the end of the AMI's name
-export TF_VAR_ami_name="gravitational-teleport-ami-ent-4.3.6"
+export TF_VAR_ami_name="gravitational-teleport-ami-ent-4.3.7"
```
Comment (benarent) [inline examples/aws/terraform/ha-autoscale-cluster/README.md:49]:
```suggestion
export TF_VAR_ami_name="gravitational-teleport-ami-ent-4.0.0"
```

**Review by benarent — COMMENTED** [review]
Looks like we've still some 3.4.7 references ( or is that intentional since it's a backport )

**Comment by webvictim** [comment]
I'll rebase this so it picks up the updated listings. In general we don't bother keeping this stuff updated in release branches though because the docs are cut off `master` and Terraform links there etc.

**Comment by webvictim** [comment]
Think this is good now (I'm not going to bother merging 4.4.1 in here as there's no intention to update the AMI IDs any further on this branch anyway)

**Review by benarent — APPROVED** [review]
Looks good.. but we are now on 4.4.2 :-)

**Review by russjones — APPROVED** [review]

**Pull request merged**